atomicapp/run.py

```diff
@@ -110,8 +113,12 @@ def _applyTemplate(self, data, component):
             except KeyError as ex:
                 name = ex.args[0]
                 logger.debug("Artifact contains unknown parameter %s, asking for it", name)
-                config[name] = self.utils.askFor(name, {"description": "Missing parameter '%s', provide the value or fix your %s" % (name, MAIN_FILE)})
+                try:
+                    config[name] = self.utils.askFor(name, {"description": "Missing parameter '%s', provide the value or fix your %s" % (name, MAIN_FILE)})
+                except EOFError:
```

Comment:
Why do we need to check and log same exception twice? (here and https://github.com/projectatomic/atomicapp/pull/169/files#diff-1e107d733a5bd148a19592d4709e33a8R138)